src/EncompassRest/Loans/RateLocks/RequestIndicators.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.RateLocks
+{
+    /// <summary>
+    /// RequestIndicators
+    /// </summary>
+    public enum RequestIndicators
+    {
+        /// <summary>
+        /// HideLog
+        /// </summary>
+        [EnumMember(Value = "hideLog")]
+        HideLog = 0,
+        /// <summary>
+        /// FakeRequest
+        /// </summary>
+        [EnumMember(Value = "fakeRequest ")]
```

Comment:
Do these actually have a blank space at the end of their names?

Reply:
Should not have spaces. Fixed in 7dcc95c